src/components/Card/Card.tsx

```diff
@@ -149,21 +149,15 @@ export const Card = ({
                 )}
                 {titleTip}
               </span>
-              <div
```

Comment:
we no longer need to wrap the button under styled-atlaskit-button div and Button component already support directly setting dataQa

Reply:
I think SDET team still using `styled-atlaskit-button` class name for selecting the button. This could cause some error could you ask them if this is ok, before change this. please revert this if it's not ok